charters/PORTABILITY/charter.md

```diff
@@ -0,0 +1,57 @@
+# Pipeline Portability Service
+
+## Description
+
+The portability service enables the confirmation that pipelines developed for the HCA DCP can be successfully ran outside of the HCA DCP.
+
+## Objectives
+
+The portability service provides the ability to run HCA DCP pipelines 
+
+## In-scope
+
+* Programmatic authorized access to submit pipelines that will be ran to confirm they execute to completion on different computing infrastructure (execution environments)
+* Programmatic authorized access to extend the backend of the portability service to include additional execution environments
+* A UI to describe the current state of submitted pipelines and execution environments
+* Documentation to describe how to submit a backend or pipeline the system
```

Comment:
Is there a missing 'to' - as in _how to submit a backend or pipeline **to** the system_ ?

Were you planning to define _pipeline_?